src/open_inwoner/laposta/api_models.py

```diff
@@ -59,17 +66,5 @@ class Member(BaseModel):
     source_url: HttpUrl | Literal[""] | None = None
     custom_fields: dict | None = None
 
-    # Let `.dict()` handle JSON encoding
-    # Source: https://github.com/pydantic/pydantic/issues/1409#issuecomment-1381655025
-    def _iter(self, **kwargs):
-        for key, value in super()._iter(**kwargs):
-            yield key, self.__config__.json_encoders.get(type(value), lambda v: v)(
-                value
-            )
-
     class Config:
-        json_encoders = {
-            # custom output conversion for datetime
-            datetime: convert_datetime_to_iso_8601_with_z_suffix,
-            IPv4Address: str,
-        }
+        json_encoders = {**JSONEncoderMixin.Config.json_encoders, IPv4Address: str}
```

Comment:
You might as well add IPv4Address to the JSONEncoderMixin?

Reply:
You're right, I added it